05-policy-changes/policy-example.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.            "content": "How has cost eligibility changed  ",
</pre><p>Missing a "?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
me lazy and just ask the model without punctuation - I removed the extra spaces